components/page-templates/B2B/QuoteDetailsTemplate/QuoteDetailsTemplate.tsx

```diff
@@ -610,7 +610,7 @@ const QuoteDetailsTemplate = (props: QuoteDetailsTemplateProps) => {
                       QuoteStatus[status] === QuoteStatus.Completed ||
                       QuoteStatus[status] === QuoteStatus.Expired ||
                       !Boolean(quoteNameField.name) ||
-                      quote?.name === quoteNameField.name
+                      quote?.name === inputValue
```

Comment:
Give some meaningful name to inputValue, it doesn't clear which input field it is.

Reply:
Resolved